data/actions/actions.xml

```diff
@@ -26,10 +26,9 @@
 	<action itemid="2580" script="tools/fishing.lua" allowfaruse="1" />
 	<action itemid="2554" script="tools/shovel.lua" />
 	<action itemid="5710" script="tools/shovel.lua" />
-	<action itemid="7731" script="tools/rope.lua" />
-	<action itemid="10511" script="tools/tool_gear.lua" />
-	<action itemid="10513" script="tools/tool_gear.lua" />
-	<action itemid="10515" script="tools/tool_gear.lua" />
+	<action itemid="10511" script="tools/toolgear.lua" />
+	<action itemid="10513" script="tools/toolgear.lua" />
+	<action itemid="10515" script="tools/toolgear.lua" />
```

Comment:
which one is correct, `tool_gear.lua` or `toolgear.lua`?

Reply:
I've reverted this change.

Reply:
Right, `tool_gear.lua`